### PR TITLE
(DOCSP-13395): Version bump to v1.9

### PR DESCRIPTION
**Pull request by melissamahoney-mongodb**
[JIRA](https://jira.mongodb.org/browse/DOCSP-13395)

### CODE REVIEW

File: publishedbranches/docs-mongocli.yaml [hunk 1]
```diff
@@ -27,13 +29,14 @@ version:
     - '1.2'
     - '1.1'
     - '1.0'
-  stable: '1.8'
-  upcoming: '1.9'
+  stable: 'v1.9'
+  upcoming: 'v1.10'
```
Comment (melissamahoney-mongodb) [inline publishedbranches/docs-mongocli.yaml:33]: Should I just remove this line if snooty doesn't use it?

File: publishedbranches/docs-mongocli.yaml [hunk 2]
```diff
@@ -27,13 +29,14 @@ version:
...
 git:
   branches:
-    manual: 'v1.9'
+    manual: 'v1.10'
```
Comment (melissamahoney-mongodb) [inline publishedbranches/docs-mongocli.yaml:36]: Should I just remove this line if snooty doesn't use it?

File: publishedbranches/docs-mongocli.yaml (hunk 1 above)
Comment (madelinezec) [reply publishedbranches/docs-mongocli.yaml:33]: You could keep upcoming for now as Jon is considering if we want to display it in the future

File: publishedbranches/docs-mongocli.yaml (hunk 2 above)
Comment (madelinezec) [reply publishedbranches/docs-mongocli.yaml:36]: I think its fine to keep for now - we haven't rehauled these published branches files. So I think its ok to wait and just keep everything to be safe. I can always do clean up in future if we decide to formally get rid of a field

**Review by madelinezec — APPROVED** [review]

File: publishedbranches/docs-mongocli.yaml (hunk 1 above)
Comment (melissamahoney-mongodb) [reply publishedbranches/docs-mongocli.yaml:33]: Sounds good, thanks.

File: publishedbranches/docs-mongocli.yaml (hunk 2 above)
Comment (melissamahoney-mongodb) [reply publishedbranches/docs-mongocli.yaml:36]: Sounds good, thanks.

**Comment by madelinezec** [comment]
@melissamahoney-mongodb for my own context - when do you next plan on deploying mongocli?

**Pull request merged**